README.md

```diff
@@ -7,916 +7,62 @@
-
-# (Optional) A list of remote log receivers, to which service logs can be sent.
-log-targets:
-
-  <log target name>:
-
-    # (Required) Control how this log target definition is combined with
-    # other pre-existing definitions with the same name in the Pebble plan.
-    #
-    # The value 'merge' will ensure that values in this layer specification
-    # are merged over existing definitions, whereas 'replace' will entirely
-    # override the existing target spec in the plan with the same name.
-    override: merge | replace
-
-    # (Required) The type of log target, which determines the format in
-    # which logs will be sent. The supported types are:
-    #
-    # - loki: Use the Grafana Loki protocol. A "pebble_service" label is
-    #   added automatically, with the name of the Pebble service as its value.
-    type: loki
-
-    # (Required) The URL of the remote log target.
-    # For Loki, this needs to be the fully-qualified URL of the push API,
-    # including the API endpoint, e.g.
-    #     http://<ip-address>:3100/loki/api/v1/push
-    location: <url>
-
-    # (Optional) A list of services whose logs will be sent to this target.
-    # Use the special keyword 'all' to match all services in the plan.
-    # When merging log targets, the 'services' lists are appended. Prefix a
-    # service name with a minus (e.g. '-svc1') to remove a previously added
-    # service. '-all' will remove all services.
-    services: [<service names>]
-
-    # (Optional) A list of key/value pairs defining labels which should be set
-    # on the outgoing logs. The label values may contain $ENV_VARS, which will
-    # be substituted using the environment for the corresponding service.
-    labels:
-      <label name>: <label value>
-```
-
-## API and clients
-
-The Pebble daemon exposes an API (HTTP over a unix socket) to allow remote clients to interact with the daemon. It can start and stop services, add configuration layers the plan, and so on.
+You can also follow the [Getting started with Pebble tutorial](https://canonical-pebble.readthedocs-hosted.com/en/latest/tutorial/getting-started/).
 
-There is currently no official documentation for the API at the HTTP level (apart from the [code itself](https://github.com/canonical/pebble/blob/master/internals/daemon/api.go)!); most users will interact with it via the Pebble command line interface or by using the Go or Python clients.
+## Getting help
 
-The Go client is used primarily by the CLI, but is importable and can be used by other tools too. See the [reference documentation and examples](https://pkg.go.dev/github.com/canonical/pebble/client) at pkg.go.dev.
+If you need support, start with the [documentation](https://canonical-pebble.readthedocs-hosted.com/en/latest/).
 
-We try to never change the underlying HTTP API in a backwards-incompatible way, however, in rare cases we may change the Go client in a backwards-incompatible way.
+To learn more about Pebble, read the following sections:
```

Comment:
Some of these seems like duplicates of the "key features" list above. Can we combine the features into that list?

And the Hacking/contributing links could just go in the next section.

Reply:
OK, I will remove this list. I think one item here adds value - the general model link. I will add a sentence about it in the quick start part maybe.